README.md

```diff
@@ -4,13 +4,17 @@
     <a href="LICENSE"><img alt="License: GPL-2.0-only" src="https://img.shields.io/badge/license-GPLv2-blue.svg"></a> <a href="https://github.com/scs/smartmeter-datacollector/pulls"><img alt="Pull Requests Welcome" src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg"></a> <a href="https://github.com/scs/smartmeter-datacollector/pulls"><img alt="Contributions Welcome" src="https://img.shields.io/badge/contributions-welcome-brightgreen.svg"></a>
     <br />
     <img alt="Python Code Checks" src="https://github.com/scs/smartmeter-datacollector/actions/workflows/python-code-checks.yml/badge.svg?branch=master"> <a href="https://pypi.org/project/smartmeter-datacollector/"><img alt="PyPI - Python Version" src="https://img.shields.io/pypi/pyversions/smartmeter-datacollector"></a>
+    <br />
```

Comment:
Can you remove the white border around the blue EKZ logo? When using Github with the dark theme it looks quite ugly due to the white border ;-)

Reply:
done

Reply:
can you please resize the icon in the readme - make it a bit smaller? For some reason I cannot fix it

Reply:
To me it looks like `width="200"` is working properly. Do you want it even smaller? If so you have to use a smaller number (e.g. 100).

Reply:
Is the size OK for you now or does it have to be smaller?

Reply:
I would put width=120

Reply:
OK then you'll have to make the change and push a new commit. I can squash all commits together for the merge so don't worry about that.

Reply:
done, thanks!